interfaces/Portalicious/README.md

```diff
@@ -1,30 +1,46 @@
-# Portalicious
+# Portal(icious)
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli).
+The front-end for the _Humanitarian Organization_ web portal, where users can interact with the 121 Platform.
 
-## Development server
+## Configuration
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+Set all ENV-variables in a local `.env`-file, based on: [`.env.example`](.env.example)-file. Each allowed/required value is explained in the example file.
 
-## Code scaffolding
+## Dependencies in use
```

Comment:
I think I would delete this section - hard to maintain and I think the code speaks for itself

Reply:
I've used this section of the old Portal quite often as a "set of bookmarks" to point other developers to the Ionic Component-documentation etc..

There isn't such a convenient place for `PrimeNG`, unfortunately.

Reply:
And I wouldn't want to add _all_ dependencies... But just pointing to the main frameworks/component-libraries...